src/canmatrix/cmjson.py

```diff
@@ -211,13 +213,20 @@ def load(f, **options):
                     is_signed = True
                 else:
                     is_signed = False
-                newsignal = Signal(signal["name"],
+                newsignal = canmatrix.Signal(signal["name"],
                                    startBit=signal["start_bit"],
                                    size=signal["bit_length"],
                                    is_little_endian=is_little_endian,
                                    is_signed=is_signed,
                                    factor=signal["factor"],
                                    offset=signal["offset"])
+
+                if "min" in signal and signal["min"]:
+                    newsignal.min = decimal.Decimal(signal["min"])
```

Comment:
@altenky 
I hast to add "decimal.Decimal"-casting here, 
not sure why ` min = attr.ib(` from signal-class did not work.
What do I wrong here?

Reply:
```suggestion
                    newsignal.min = newsignal.float_factory(signal["min"])
```

The converter for an `attr.ib()` is only applied when set through the constructor.  We could make `@property`s which apply the converter on assignment as well but, I dunno...  I do have a thing which does that automatically.  But, if we want that it should certainly be it's own effort.

Do beware when writing json as to whether you write floats or strings.  The former loses the benefit of using `decimal.Decimal()`.

Sorry I missed the review request...